buildx/working-with-buildx.md

```diff
@@ -172,3 +200,7 @@ command called [`docker buildx bake`](../engine/reference/commandline/buildx_bak
 The `bake` command supports building images from compose files, similar to 
 [`docker-compose build`](../compose/reference/build.md), but allowing all the
 services to be built concurrently as part of a single request.
+
+There is also support for custom build rules from HCL/JSON files allowing
+better code reuse and different target groups. The design of bake is in very
+early stages and we are looking for feedback from users.
```

Comment:
@crazy-max Should we add a pointer on how users can provide feedback?

Reply:
@usha-mandya I put a link to buildx repo issues. Lgty?